src/FakeJoinCallback.php

```diff
@@ -16,7 +16,7 @@ public function getAlias(): ?string
 
     public function __call($name, $arguments)
     {
-        if ($name === 'as') {
+        if ('as' === $name) {
```

Comment:
Can we remove this rule? I'm not the biggest fan of this, and this is not part of PSR, right?

Reply:
Pretty sure is this config, but  I cannot try it for a while

```php
    'yoda_style' => [
        'equal' => false,
        'identical' => false,
        'less_and_greater' => false,
    ],
```